README.md

```diff
@@ -1,24 +1,27 @@
-# SDK for Aptos Node API
+# Typescript SDK for Aptos
 
 [![Discord][discord-image]][discord-url]
 [![NPM Package Version][npm-image-version]][npm-url]
 [![NPM Package Downloads][npm-image-downloads]][npm-url]
 
-The Aptos TypeScript SDK provides a convenient way to interact with the Aptos blockchain using TypeScript. It offers a set of utility functions, classes, and types to simplify the integration process and enhance developer productivity.
+The Aptos TypeScript SDK provides a convenient way to interact with the Aptos blockchain using TypeScript. It offers a 
+set of utility functions, classes, and types to simplify the integration process and enhance developer productivity.
+
+This repository supports version >= 2.0.0 of the [Aptos SDK npm package](https://www.npmjs.com/package/aptos).
 
 ## Installation
 
 ##### For use in Node.js or a web application
 
-```ts
-pnpm install aptos
+Install with your favorite package manager such as npm, yarn, or pnpm:
+```bash
+npm install aptos
```

Comment:
fyi, we might be publishing the new sdk under the `@aptos-labs` namespace to be consistent with all the other tools. 
Also, we might also ship the alpha version with an `@experimental` tag to let devs know this is not a production ready version and API/interface can get changed

Reply:
Let's update it when we get there :)